src/stories/decorators.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+/**
+ * Decorator to replicate the insane WP-admin dashboard class/ID structure.
+ *
+ * @param storyFn
+ */
+export const DashboardAdminDecorator = (storyFn) => (
+	<div className="wp-admin" id="wpbody-content">
+		<div className="metabox-holder" id="dashboard-widgets" style={ { padding: 20 } }>
```

Comment:
20px?

Reply:
React automatically converts integer values into `px` so shouldn't need to be explicit here.